d3a_interface/constants_limits.py

```diff
@@ -122,6 +122,8 @@ class WindSettings:
         MAX_WIND_TURBINE_OUTPUT_W = 160
 
     class IAASettings:
+        GRID_FEE_TYPE = 1
```

Comment:
Reason for not using an enum is because you cannot json-serialize it, right ?

Reply:
yes. I dont like the int either, but it is the same for the market_type.